Makefile

```diff
@@ -58,3 +66,15 @@ archive: build
 	@tar --transform "s|^./|frostfs-contract-$(VERSION)/|" \
 		-czf frostfs-contract-$(VERSION).tar.gz \
 		$(shell find . -name '*.nef' -o -name 'config.json')
+
+# Package for Debian
+debpackage:
+	dch --package neofs-contract \
```

Comment:
What about changing this during rebase? It is a nice property -- being able to checkout on any commit and everything works.

Reply:
I can fixup https://github.com/TrueCloudLab/frostfs-contract/pull/6/commits/51de0c1309bfb762bdf41832ea095dca5192d462 with https://github.com/TrueCloudLab/frostfs-contract/pull/6/commits/bceab4e1330820e5bab62f4da33d8b3e432e07a1

Thought that it would be easier to see deb packages changes in separate commit.

Reply:
@fyrchik Done.